utils/scripts/01_create_tables_associations.sql

```diff
@@ -1,37 +1,265 @@
 DROP DATABASE IF EXISTS farmly;
-CREATE DATABASE farmly;
 
-\c farmly;
+CREATE DATABASE farmly WITH OWNER = root ENCODING = "UTF8" TABLESPACE = pg_default CONNECTION LIMIT = - 1;
 
+\c farmly
+-- SEQUENCE: public.addresses_id_seq
+CREATE SEQUENCE IF NOT EXISTS public.addresses_id_seq
+INCREMENT 1 START 1
+MINVALUE 1
+MAXVALUE 2147483647
+CACHE 1;
```

Comment:
esto hace algo ?

Reply:
el cache 1

Reply:
eso me lo diste tú jajaja en el script que me diste de la base de datos.
Según google;
The optional clause CACHE cache specifies how many sequence numbers are to be preallocated and stored in memory for faster access. The minimum value is 1 (only one value can be generated at a time, i.e., no cache), and this is also the default. CYCLE.